documentation/source/building.rst

```diff
@@ -128,6 +128,10 @@ and
 
       Use to add additional dependencies to the simulation target.
 
+.. make:var:: COCOTB_NVC_TRACE
+
+      Set this to 1 to enable display of VHPI traces when using the NVC VHDL simulator.
+
```

Comment:
Something bad has happened with your rebase, you seem to have reverted all our recent changes to this file.